Declaring apollo-link as a dev dependency causes complaints in typescript

Caused by #445, the dependency is broken.

As `apollo-link` is now a dev dependency cause the following typescript error since it is not installed by default.

```
node_modules/graphql-tools/dist/stitching/introspectSchema.d.ts(2,28): error TS2307: Cannot find module 'apollo-link'.
node_modules/graphql-tools/dist/stitching/makeRemoteExecutableSchema.d.ts(1,28): error TS2307: Cannot find module 'apollo-link'.
```

Adding `apollo-link` as a dev dependency (suggested in #424) resolves the above error, but causes the following typescript error in it's place.
```
node_modules/apollo-link/lib/linkUtils.d.ts (1,23): error TS2688: Cannot find type definition file for 'zen-observable'.
node_modules/apollo-link/lib/types.d.ts (1,23): error TS2688: Cannot find type definition file for 'zen-observable'.
node_modules/apollo-link/lib/link.d.ts (1,23): error TS2688: Cannot find type definition file for 'zen-observable'.
```

--- a/package.json
+++ b/package.json
@@ -49,6 +49,7 @@
   "homepage": "https://github.com/apollostack/graphql-tools#readme",
   "dependencies": {
     "apollo-utilities": "^1.0.1",
+    "apollo-link": "^1.0.0",
     "deprecated-decorator": "^0.1.6",
     "uuid": "^3.1.0"
   },
@@ -62,7 +63,6 @@
     "@types/node": "^8.0.47",
     "@types/uuid": "^3.4.3",
     "@types/zen-observable": "^0.5.3",
-    "apollo-link": "^1.0.0",
     "body-parser": "^1.18.2",
     "chai": "^4.1.2",
     "express": "^4.16.2",

